### PR TITLE
6S1: Implement TTS model

### PR DESCRIPTION
**Pull request by isabellemyz**
- implemented TTS model (openai) on frontend and backend
- moved some functions (`getResponse`, `getResponseAudio`, `refresh`) into sep file - part of larger initiative to clean up code structure
- cleaned up some unused imports
- loom link: https://www.loom.com/share/0ad2bebac3d64bdc965a061a26e7f2ef?sid=e7f642df-3d68-4012-92fa-691387cb46be

### CODE REVIEW

File: backend/processing/model_input.py [hunk 1]
```diff
@@ -32,5 +29,28 @@ def get_response(user_input, client):
     
     return response.choices[0].message.content
 
+def get_response_audio(text, client):
+    try:
```
Comment (khoamluong) [inline backend/processing/model_input.py:33]: pass voice parameter to get_response_audio maybe, eventually we want to have users choose voice dyanmically

Comment (TanujKannan) [reply backend/processing/model_input.py:33]: Yeah I agree. Additionally, I think the whole body:
        audio_response = client.audio.speech.create(
            model="tts-1",
            voice="nova",
            input=text
        )
comes from a config file. Something like model = config.ttsmodel, voice = config.ttsmodel.voice, etc.

**Review by TanujKannan — APPROVED** [review]
Looks good, need to create "Make config file" ticket.

File: backend/fast_api.py [hunk 2]
```diff
@@ -75,6 +75,14 @@ async def get_response_endpoint(request: TextOutput):
     response = get_response(request.text, client)
     return ModelResponse(response=response)
 
+@app.post("/get_response_audio")
+async def get_response_audio_endpoint(request: TextOutput):
```
Comment (khoamluong) [inline backend/fast_api.py]: - add a request type to STTRequest or name something different than TextOutput cuz not meaningful and add an optional field for voice maybe like voice: Optional[str] = Field(default="nova") or something 
- we can add to voice later

**Review by khoamluong — APPROVED** [review]

**Pull request merged**